postgres/pgdump.go

```diff
@@ -585,6 +620,10 @@ func updateSchema(conv *internal.Conv, table string, cs []constraint, stmtType s
 			ct := conv.SrcSchema[table]
 			ct.ForeignKeys = append(ct.ForeignKeys, toForeignKeys(c)) // Append to previous foreign keys.
 			conv.SrcSchema[table] = ct
+		case nodes.CONSTR_UNIQUE:
```

Comment:
I think we are missing tests for this case.

Reply:
added tests for this case